apps/studio/src/common/LocalStorage.ts

```diff
@@ -68,6 +68,13 @@ export const SmartLocalStorage = {
         expiry: now.getTime() + ttl,
       }
     localStorage.setItem(key, JSON.stringify(item))
-  }
+  },
+  /** Returns `true` if the key exists */
+  exists(key: string) {
+    return localStorage.getItem(key) !== null;
```

Comment:
not a hill to die on, but I'd have the check as `!= null` just in case undefined gets added there for some reason.